assets/js/admin/orders/order-overview/models/state.js

```diff
@@ -177,7 +145,7 @@ export const State = Backbone.Model.extend(
 				return this.get( 'order' ).total;
 			}
 
-			return this.getSubtotal() + this.getTax();
+			return ( this.getSubtotal() - this.getDiscount() ) + this.getTax();
```

Comment:
Sorry to keep doing this one thing at a time. Currently, the subtotal is showing the combined amount of the products. Discounts are applied to modify the total.

However, other adjustments are not being calculated (fees/credits) into the total. If I go ahead and create the order, the fees and credits are displayed, but the order total is using what showed when I was creating the order (incorrect):

![image](https://user-images.githubusercontent.com/4853066/115743111-5e0c0a00-a35f-11eb-81a4-0e99b80604f8.png)

Reply:
@robincornett I see, I think https://github.com/easydigitaldownloads/easy-digital-downloads/pull/8581/files#diff-e8e2a669744f756eb93f2fbecb659dd2eb08b6662460386f46dbe43c6ff17cc5L101-L118 needs to be moved to the total calculation.

Question: are taxes calculated after all adjustments (discount, credit, fee) applied? So your screenshot should have a $3 total, and if taxes enabled a tax added to $3?

Reply:
I think that's complicated. It looks like the tax calculations are handled in the cart and then passed for each order item to `edd_build_order`, so I don't think tax calculations are handled there, just storage. In the cart, we have:

https://github.com/easydigitaldownloads/easy-digital-downloads/blob/6a09489eabefde58bf56e0981c139ba0c2677f23/includes/cart/class-edd-cart.php#L240-L258

So I think the math goes like this for each item:
1. item subtotal
2. plus (item) fees
3. minus discount
4. calculate and add tax
5. then subtract tax if taxes are included in the price

Then those are added together for the cart subtotal (which is different in the cart than it is in the order, I think, which is weird), and the total becomes:
1. subtotal of all items (before discounts)
2. get any order level fees
3. add tax to fees if they are taxed
4. subtotal, subtract discounts, add the calculated cart tax, and add the calculated fees (which may be taxed)

Fees can be taxed, or not, so that adds a layer of complexity.

Actually some of this is replicated in `edd_build_order`, at least outside of the individual item tax amount. My assumption is that this stage of the manual order, though, needs to follow the cart logic?

Reply:
Also with credits being totally new in 3.0, I'm not sure on the answer to that. This might be a question for someone financial?

Reply:
> add tax to fees if they are taxed

For manual Order creation we don't have UI in place to facilitate the [`no_tax`](https://docs.easydigitaldownloads.com/article/543-edd-fees) argument normally used with preregistered fees. Do we need a checkbox that the fee amount shouldn't be taxed?

cc @ashleyfae